wolfssl/wolfcrypt/types.h

```diff
@@ -480,6 +480,14 @@
                 #define XSTRTOK(s1,d,ptr) strtok_r((s1),(d),(ptr))
             #endif
         #endif
+
+        #if defined(WOLFSSL_CERT_EXT) || defined(HAVE_OCSP) || \
+            defined(HAVE_CRL_IO) || defined(HAVE_HTTP_CLIENT) || \
+            !defined(NO_CRYPT_BENCHMARK)
+
+            #include <stdlib.h>
+            #define XATOI(s)          atoi((s))
```

Comment:
No way for the user to override and sodlib.h must exist?  I think we need to follow the pattern of XMEMSET and others.

Reply:
Fixed. Added ability to define your own `XATOI`. It can also be overridden using `STRING_USER`.